charts/datadog/values.yaml

```diff
@@ -374,6 +374,10 @@ datadog:
     # datadog.networkMonitoring.enabled -- Enable network performance monitoring
     enabled: false
 
+  serviceMonitoring:
```

Comment:
maybe, It can be good to add a comment with a link to the official doc.

Reply:
USM is in private beta right now and we don't have official documentation yet.

Reply:
Ok, so could you add a comment saying it is in private beta, please contact datadog if you are interested (or something like that)

Reply:
Sure, will do.

Reply:
Done.